src/main/java/com/liferay/faces/util/logging/LoggerFactory.java

```diff
@@ -33,39 +30,33 @@
 
 	// Private Constants
 	private static final String CLASS_NAME_LOG4J_LOGGER = "org.apache.log4j.Logger";
-
-	// Statically-Initialized Private Constants
-	private static boolean LOG4J_AVAILABLE = false;
+	private static final boolean LOG4J_AVAILABLE;
 
 	static {
 
+		boolean log4jAvailable = true;
+
 		try {
 			Class.forName(CLASS_NAME_LOG4J_LOGGER);
-			LOG4J_AVAILABLE = true;
 
 			try {
-				new LoggerLog4JImpl(CLASS_NAME_LOG4J_LOGGER);
+				LogManager.getLogger(CLASS_NAME_LOG4J_LOGGER);
 			}
 			catch (NoClassDefFoundError e) {
```

Comment:
@stiemannkj1 I think you are correct. The catch for NoClassDefFound could be moved to the try/catch block inside of the LoggerLog4JImpl constructor.